### ISSUE TITLE
Reading the outputs from the Layer package

### ISSUE THREAD

**Issue by liu-ran** [MITgcm/xmitgcm#270]
When reading the outputs from the Layer package, there is a KeyError here as shown in the screenshot. Doesn't the _xmitgcm_ read the output from the Layer package?

![微信图片_20210629121417](https://user-images.githubusercontent.com/48266323/123736481-c59c8380-d8d3-11eb-9433-268e4b7fa27f.png)

**Comment by rabernat** [comment]
Yes, it should be able to read from the layers file. Are you sure that the `.meta` files are all present in your directory?

**Comment by NelsonPoumaere** [comment]
Hi, in case that might help anyone landing on this issue, be sure to include the available_diagnostics.log file in the directory containing the data: I ran into the same problem as above and this did the trick. Got this clue from https://github.com/MITgcm/xmitgcm/issues/173.
